client/packages/system/src/Immunisation/VaccineCourseView/VaccineCourseView.tsx

```diff
@@ -319,21 +260,26 @@ export const VaccineCourseView: FC = () => {
               value={buffer}
               fullWidth
               onBlur={() => {
-                onUpdate({ numberOfDoses: value });
+                updatePatch({ doses: value });
```

Comment:
This isn't getting saved... I think calling updatePatch here, and then again inside updateSchedule is overwriting this one? Maybe you can do both updates in the one updatePatch inside updateSchedule?

If set doses to 4, save then refresh, get this:
![Screenshot 2024-06-14 at 2 15 13 PM](https://github.com/msupply-foundation/open-msupply/assets/55115239/f4414ad0-f423-4145-953d-d4ee3700d818)

Makes me wonder why we don't just derive # doses from the number of vaccine course schedules though?